examples/lang/erc20/Cargo.toml

```diff
@@ -5,9 +5,12 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-ink_core = { path = "../../../core" }
-ink_model = { path = "../../../model" }
-ink_lang = { path = "../../../lang" }
+ink_core = { git = "https://github.com/paritytech/ink/", package = "ink_core" }
```

Comment:
We should not depend on `node-runtime-types` via ink! which enforces these GitHub dependencies and instead copy those important examples over to that repo. Generally we want to avoid GitHub dependencies to crates within our own repo.